### PR TITLE
#205 - Documentation for active learning

### PR DESCRIPTION
**Pull request by matchlesswei**
- Documentation for active learning sidebar

### CODE REVIEW

File: inception-active-learning/src/main/resources/META-INF/asciidoc/user-guide/annotation_activeLearning.adoc [hunk 1]
```diff
@@ -0,0 +1,47 @@
+[[sect_annotation_recommendation]]
+=== Active Learning
+Once the recommenders are set in the <<sect_projects_recommendation, Project Settings>>, and assuming
```
Comment (Rentier) [inline inception-active-learning/src/main/resources/META-INF/asciidoc/user-guide/annotation_activeLearning.adoc]: I would start with explaining what active learning actually does. From the paper:

> Active learning is a family of methods which seeks to optimize the learning rate of classification algorithms by intelligently soliciting labels from a human user for which the system only has low confidence. This means that recommenders can make better suggestions with less user interactions, allowing the user to perform quicker and more accurate annotations.

**Comment by reckart** [comment]
The HTML version is what we really use :) The PDF version is built, but we do not link it anywhere. It might be that the formatting in the PDF version is off in some places.

File: inception-active-learning/src/main/resources/META-INF/asciidoc/user-guide/annotation_activeLearning.adoc [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+image::activeLearning1.png[align="center"]
+
+One can now click the active learning icon on the left side and the Active Learning sidebar shows up
+. One can now select the layer like POS layer for annotation and click “Start” for starting an
+active learning session:
+
+image::activeLearning2.png[select, 350, 350, align="center"]
+
+The Active Learning sidebar will then start showing recommendations, one by one, according to the
+“uncertainty sampling” learning strategy. For every recommendation, it shows the related text, the
```
[the remark this answers is not in the document]
Comment (matchlesswei) [reply inception-active-learning/src/main/resources/META-INF/asciidoc/user-guide/annotation_activeLearning.adoc]: If we leave it out, would users have the question why this kind of order?

File: inception-active-learning/src/main/resources/META-INF/asciidoc/user-guide/annotation_activeLearning.adoc [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+suggested annotation, the confidence score and a delta that represents the difference between the
+given score and the closest score calculated for another suggestion made by the same recommender to
+that text. The recommendation is also highlighted in the central annotation editor.
+
+One can now “Accept”, “Reject” or “Skip” this recommendation in the Active Learning sidebar:
```
[the remark this answers is not in the document]
Comment (matchlesswei) [reply inception-active-learning/src/main/resources/META-INF/asciidoc/user-guide/annotation_activeLearning.adoc]: I prefer mentioning the learning history here to tell users that these actions will be recorded and they can find them. This is also what happened after pressing the button.

One can now “Accept”, “Reject” or “Skip” this recommendation in the Active Learning sidebar:
The acceptance, rejection or skipping will be recorded and displayed in the learning history of the Active Learning sidebar. After a suggestion is accepted, the text is annotated with that recommended annotation. If the user rejects a suggestion ...

File: inception-active-learning/src/main/resources/META-INF/asciidoc/user-guide/annotation_activeLearning.adoc [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+Eventually, they could be shown again at the end of the active learning session, when there are no
+more unanswered suggestions.
+
+image::activeLearning4.png[align="center"]
+
+After the user takes an action on the current suggestion, the next recommendation will then show up
+in the sidebar, and the central annotation editor will jump to its corresponding location (which
+could sometimes be in another document).
+
+The learning history contains a log of all the actions that were taken by the user regarding the
```
[the remark this answers is not in the document]
Comment (matchlesswei) [reply inception-active-learning/src/main/resources/META-INF/asciidoc/user-guide/annotation_activeLearning.adoc]: This paragraph is to describe the learning history in detail.

**Comment by matchlesswei** [comment]
It looks good in HTML version. But why in the last sentence, there is "<<<"? 

screen shot 2018-05-16 at 14 10 52

**Pull request merged**